src/js/containers/search/filters/AgencyContainer.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+  * AgencyContainer.jsx
+  * Created by Emily Gullo 11/30/16
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as searchFilterActions from 'redux/actions/search/searchFilterActions';
+
+import Agency from 'components/search/filters/agency/Agency';
+
+const propTypes = {
+    updateSelectedFundingAgencies: React.PropTypes.func,
+    updateSelectedAwardingAgencies: React.PropTypes.func
+};
+
+export class AgencyContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        // Bind functions
+        this.selectAgency = this.selectAgency.bind(this);
+        this.removeAgency = this.removeAgency.bind(this);
+    }
+
+    selectAgency(agency, isValid, agencyType) {
```

Comment:
Good call - combined these into one `toggleAgency` function. I'm keeping the function in `AgencyContainer`, as it's passed down through it's child, `Agency`, which passes it to multiple grandchildren – `AgencyListConatiner` and `SelectedAgencies`. I'd prefer not to duplicate it in both of those, which would also involve converting `SelectedAgencies` to a container.